numba/tests/test_comprehension.py

```diff
@@ -0,0 +1,256 @@
+from __future__ import print_function
```

Comment:
This sounds fine noting it will likely be the same code path backing them. I'm looking forward to "array comprehensions" :)